src/app/page.tsx

```diff
@@ -1,11 +1,26 @@
-import Image from "next/image";
-import Header from "@/components/header";
-import styles from "./page.module.css";
+import Header from "@/components/Header/Header";
+import Banner from "@/components/Banner/Banner"
+import { Contacts } from "@/components/Contacts/contacts";
+import styles from "./page.module.scss";
+import PromItems from "@/components/Items/promItems";
+import Categories from "@/components/Product categories/categories";
+import { Footer } from "@/components/Footer/footer";
+import Slider from "@/components/Slider/slider"
+import Sponsor from "@/components/Sponsor/sponsor"
```

Comment:
```suggestion
import PromItems from "@/components/Items/promItems";
import Categories from "@/components/Product categories/categories";
import { Footer } from "@/components/Footer/footer";
import Slider from "@/components/Slider/slider";
import Sponsor from "@/components/Sponsor/sponsor";
import styles from "./page.module.scss";
```
Стили подключаются в самом конце.